bash-job.yml

```diff
@@ -20,18 +20,16 @@ spec:
               HOME=/tmp pip install --user virtualenv
               PYTHONPATH="/tmp/.local/lib/python3.6/site-packages:$PYTHONPATH" /tmp/.local/bin/virtualenv  /tmp/v
               source /tmp/v/bin/activate
-              pip install mwclient
-              pip install ansible
-              pip install Jinja2
+              pip install -r /tmp/fedora-ci-stats/requirements.txt
```

Comment:
At this time repo is not cloned.
Clone is done a few lines bellow. no requirements.txt so far.

Reply:
Fixed